### PR TITLE
Add delete log group function

### PR DESCRIPTION
**Pull request by yingzhanredhat**

### CODE REVIEW

File: pkg/aws_client/aws_v1/cloudwatch_log.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ func (client *AWSClient) DescribeLogGroupsByName(logGroupName string) (cloudwatc
 		LogGroupNamePrefix: &logGroupName,
 	})
 	if err != nil {
-		fmt.Println("Got error describe log group: ", err)
+		fmt.Printf("Got error describe log group:%s ", err)
```
Comment (xueli181114) [inline pkg/aws_client/aws_v1/cloudwatch_log.go]: Don't use fmt.Println or  fmt.Printf. Use log please

File: pkg/aws_client/aws_v1/cloudwatch_log.go [hunk 2]
```diff
@@ -21,7 +21,17 @@ func (client *AWSClient) DescribeLogStreamByName(logGroupName string) (cloudwatc
 		LogGroupName: &logGroupName,
 	})
 	if err != nil {
-		fmt.Println("Got error describe log stream: ", err)
+		fmt.Printf("Got error describe log stream: %s", err)
```
Comment (xueli181114) [inline pkg/aws_client/aws_v1/cloudwatch_log.go]: Don't use print. Let's use log.

File: pkg/aws_client/aws_v1/cloudwatch_log.go [hunk 3]
```diff
@@ -21,7 +21,17 @@ func (client *AWSClient) DescribeLogStreamByName(logGroupName string) (cloudwatc
...
+	}
+	return *output, err
+}
+
+func (client *AWSClient) DeleteLogGroupByName(logGroupName string) (cloudwatchlogs.DeleteLogGroupOutput, error) {
+	output, err := client.cloudWatchLogsClient.DeleteLogGroup(&cloudwatchlogs.DeleteLogGroupInput{
+		LogGroupName: &logGroupName,
+	})
+	if err != nil {
+		fmt.Printf("Got error delete log group: %s", err)
```
Comment (xueli181114) [inline pkg/aws_client/aws_v1/cloudwatch_log.go]: Same to above

**Comment by yingzhanredhat** [comment]
@xueli181114 Would you help review again?

**Pull request merged**